.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Tell us about a problem with the project's current functionality
+title: "[Bug]"
+labels: bug
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the bug you're experiencing in the Title above -->
+
+
+## Detailed Description
+<!--- Provide a detailed description of the bug you are experiencing -->
+
+
+## To Reproduce
+<!--- Describe the steps that lead to experiencing the bug -->
+
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in, such as the following bullet points -->
+* Version used:
+* Environment name and version (e.g. Chrome 39, node.js 5.4):
+* Operating System and version (desktop or mobile):
+
```

Comment:
Can we add a priority query here? Something like:

Please indicate whether fixing this bug is high, medium, or low priority for you. If the issue is time-sensitive for you, please let us know when you need it addressed by.